main.py

```diff
@@ -19,7 +19,7 @@ async def on_ready():
 
 async def timer():
     while True:
-        await asyncio.sleep(1)
+        await asyncio.sleep(60)
```

Comment:
Udělal bych to podle Milana spíš

Reply:
Doing something at a specific time each day:

```
import datetime
from discord.ext import commands, tasks

utc = datetime.timezone.utc

# If no tzinfo is given then UTC is assumed.
time = datetime.time(hour=8, minute=30, tzinfo=utc)

class MyCog(commands.Cog):
    def __init__(self, bot):
        self.bot = bot
        self.my_task.start()

    def cog_unload(self):
        self.my_task.cancel()

    @tasks.loop(time=time)
    async def my_task(self):
        print("My task is running!")
```

Tohle je z toho tasks modulu v discordpy. Melo by to kazdy den v 8:30 poslat zpravu